test_requirements.txt

```diff
@@ -1,14 +1,15 @@
-ipython
 bpython
+coveralls
```

Comment:
If you're removing coveralls from this pull request, you should remove it from this file as well.